app/src/main/java/com/example/harvestfresh/LoginActivity.java

```diff
@@ -21,15 +22,22 @@ public class LoginActivity extends AppCompatActivity {
     private EditText etUsername;
     private EditText etPassword;
     private Button btnLogin;
+    private Button btnSignup;
+    final FragmentManager fragmentManager = getSupportFragmentManager();
```

Comment:
In java, the default privacy level is package-private which is higher than private, but lower than public. Please move this appropriately.

Also, put a new line between each privacy type so the difference is easier to read:


public static final

final

private
private
private
private

Reply:
Done! Sorry I wrote this code before I saw your previous comment abt variable orders

Reply:
No worries, just try to revise whatever you can when youre working when someone makes a comment on something else, just to prevent getting the same comments again :)